presentation/src/main/AndroidManifest.xml

```diff
@@ -39,7 +39,7 @@
 
     <application
         android:name=".common.QKApplication"
-        android:allowBackup="true"
+        android:allowBackup="false"
```

Comment:
This one syncs the database/preferences to Google cloud which won't make sense in most of the app instances. We can remove this if you think otherwise.